docs/faq.md

```diff
@@ -0,0 +1,71 @@
+# F.A.Q.
+1. What happened to prometheus-postgresql-adapter/pg_prometheus?
+
+    Promscale and [Promscale extension](https://github.com/timescale/promscale_extension) are replacing those products with improved features and performance.
+
+2. Does Promscale work with vanilla PostgreSQL?
+
+    Yes, Promscale works with PostgreSQL without the TimescaleDB extension installed. However, we suggest running it with TimescaleDB to improve the general performance and reduce storage size using the compression feature included with TimescaleDB. 
+
+3. Is Promscale compatible with Kubernetes?
+
+    Yes. Promscale has first-class Kubernetes support. You can actually use the tobs tool to easily include it in your observability stack: https://github.com/timescale/tobs ([demo video](https://www.youtube.com/watch?v=MSvBsXOI1ks))
+
+4. Can I run PromQL queries directly on Promscale?
+
+    Yes. Promscale implements the [HTTP API](https://prometheus.io/docs/prometheus/latest/querying/api/) from Prometheus to support the full PromQL query interface. You can use it as a Prometheus datasource in 3rd party tools like Grafana.
+
+5. How do I query the data using SQL?
+
+    There is a document describing the SQL schema and API that Promscale exposes [here](https://github.com/timescale/promscale/blob/master/docs/sql_schema.md).
+
+6. How do I delete data?
+
+    Right now, there are two ways of deleting ingested data:
+        - Automatically using the data retention properties ([details here](https://github.com/timescale/promscale/blob/master/docs/sql_schema.md#data-retention))
+        - Manually through SQL interface (either Postgres [truncate table](https://www.postgresql.org/docs/current/sql-truncate.html) or [drop_chunks](https://docs.timescale.com/latest/api#drop_chunks) TimescaleDB function)
+
+7. What is the best way to set up HA (high availability) on Promscale?
+
+    You can find the documentation on Promscale HA setup [here](https://github.com/timescale/promscale/blob/master/docs/high-avaliability/prometheus-HA.md).
+
+8. What is this Promscale extension for?
+
+    Promscale extension is an additional extension separate from TimescaleDB that can be used to improve the general performance characteristics of Promscale. You can find more info here: https://github.com/timescale/promscale_extension
```

Comment:
Since I have seen a significant number of users asking on how to install Promscale extension. should we mention here that Promscale extension is installed by Promscale on the first run by itself, in case the extension is not installed then.